### ISSUE TITLE
[Dev-Enhancement] CocoaScript ESLint Plugin

### ISSUE THREAD

**Issue by booc0mtaco** [sketch-hq/SketchAPI#869]
This is a little different than most enhancement requests, but figured this was the best place to ask. A few years ago, I created a very minimal ESLint plugin to support a few aspects of CocoaScript ([here](https://www.npmjs.com/package/eslint-plugin-cocoascript)). Before I got to a place where I wanted to add more functionality, my project ended and I left it as-is.

Fast forward a few years (5!) and it seems the JS-based API is more fully fleshed out than when I was doing more sketch plugin development. So first question, *is using CocoaScript for plugin development still recommended*? If not, then I will likely archive my [plugin repo](https://github.com/booc0mtaco/eslint-plugin-cocoascript). 

If so, then would there be some benefit / desire in completing this ESLint plugin? I didn't get a sense from the [forums](https://sketchplugins.com/t/plugins?sort=latest&q=cocoascript) whether people are using the objc syntax often, or mostly using the JS-equivalents.

tl;dr Basically, I am trying to determine if I should archive this repository, or maybe hand it off in case having such a plugin has any value.

Let me know your thoughts!
